packages/server/graphql/types/Threadable.ts

```diff
@@ -67,8 +68,16 @@ const Threadable = new GraphQLInterfaceType({
   name: 'Threadable',
   description: 'An item that can be put in a thread',
   fields: threadableFields,
-  resolveType: ({status}) => {
-    return status ? Task : Comment
+  resolveType: ({status, options}) => {
```

Comment:
might be worthwhile casting to the `database/Threadable.ts` type so we get a little type safety here. 
then again, you could just merge that file into this one

Reply:
@mattkrick I've decided to leave `database/Threadable.ts` in a separate file as there's more than one place that uses this type and in my opinion, it's cleaner that way. I've added type checks, though. Let me know what do you think!